packages/dev/gui/src/3D/controls/button3D.ts

```diff
@@ -17,13 +17,21 @@ export class Button3D extends AbstractButton3D {
     /** @internal */
     protected _currentMaterial: Material;
 
+    protected _width: number;
+    protected _height: number;
+    protected _depth: number;
+
     /**
      * Creates a new button
      * @param name defines the control name
      */
-    constructor(name?: string) {
+    constructor(name?: string, width = 1, height = 1, depth = 0.08) {

```

Comment:
not sure how much it fits in this usecase, but an `options` object will allow us to expand the parameters passed to the constructor without making it very long. Think it makes sense here?